var/spack/repos/builtin/packages/gcc-runtime/package.py

```diff
@@ -44,9 +47,15 @@ class GccRuntime(Package):
         "ubsan",
     ]
 
+    # libgfortran ABI
+    provides("fortran-rt", "gfortran")
+    provides("gfortran@3", when="%gcc@:6")
+    provides("gfortran@4", when="%gcc@7")
+    provides("gfortran@5", when="%gcc@8:")
+
     def install(self, spec, prefix):
         if spec.platform in ["linux", "cray", "freebsd"]:
-            libraries = self._get_libraries_elf()
+            libraries = get_elf_libraries(compiler=self.compiler, libraries=self.LIBRARIES)
```

Comment:
what's this change about?

Reply:
This is just to reuse the same code in `intel-oneapi-runtime`. It's temporary, and I think in the end we should probably move this somewhere in core Spack (my guess is that it will be used by most runtimes).